README.md

```diff
@@ -85,7 +85,8 @@ After that, you can use the `Restore`, `Build`, `Pack` targets to restore packag
 
 #### Important to Note
 
-- It will only work with Visual Studio IDE (Windows/Mac) as it requires the desktop `msbuild` and the target Platform SDKs which are not cross platform.
+- It will only work with an IDE that uses the desktop `msbuild` (i.e. Visual Studio) and the target Platform SDKs which are not cross platform.
+- When using JetBrains Rider, you will need to point to your desktop MSBuild in your settings (Settings > Build, execution, deployment > Use MSBuild Version.) You must be using Rider on Windows.
```

Comment:
Any reason Rider on Mac wouldn't work?

Reply:
Rider on Mac and Linux use the .NET Core SDK MSBuild - the one that this SDK doesn't work on - whereas Rider on Windows can be pointed to the desktop msbuild.

Reply:
Rider on Mac can't be pointed at msbuild.exe? It's there since VSfM uses it.

Reply:
Oh, well then maybe it can 🤷‍♂️ Well if this SDK works with VSfM, then I'm pretty sure it'll work with Rider, as you can specify the MSBuild location in any Rider installation.